src/common/constants/endpoints.ts

```diff
@@ -3,3 +3,5 @@ import { env } from '../../types/env';
 const domain = (env && env.REACT_APP_ALKEMIO_DOMAIN) ?? '';
 export const publicGraphQLEndpoint = domain + '/api/public/graphql';
 export const privateGraphQLEndpoint = domain + '/api/private/graphql';
+
+export const geoEndpoint = domain + '/api/public/rest/geo';
```

Comment:
Please expose those endpoint through a setting + configuration in the configuration API.